examples/mdb/mongodbcluster.yaml

```diff
@@ -16,10 +16,9 @@ spec:
   forProvider:
     name: example-mongodb
     environment: PRESTABLE
-    networkIdRef:
-      name: example-net
+    networkId: <your-network-id>
     clusterConfig:
-      - version: "4.2"
+      - version: "4.4" # required either [4.4 4.2 4.0 3.6 5.0-enterprise 4.4-enterprise 5.0]
```

Comment:
It's better to provide link on some site where supported versions described. It will always be updated without commits in repo

Reply:
Unfortunately, I didn't find the appropriate link for it. That's why I deleted these comments.